cpp/velox/memory/velox_memory_pool.h

```diff
@@ -27,6 +27,6 @@ constexpr int64_t kMaxMemory = facebook::velox::memory::kMaxMemory;
 
 std::shared_ptr<facebook::velox::memory::MemoryPool> AsWrappedVeloxMemoryPool(MemoryAllocator* allocator);
 
-std::shared_ptr<facebook::velox::memory::MemoryPool> GetDefaultWrappedVeloxMemoryPool();
+facebook::velox::memory::MemoryPool* GetDefaultWrappedVeloxMemoryPool();
```

Comment:
I see.

because the life of `DefaultWrapperVeloxMemoryPool` is the same as the process, so it's no need to manage it with the smart pointer.

we can keep this change and modify it to satisfy the demand if we call the Velox-related interfaces taking `std::shared_ptr<MemoryPool>`.

Reply:
Fair enough. Thanks.